src/common/cudnn_cxx.cc

```diff
@@ -0,0 +1,336 @@
+/*
```

Comment:
I think it would be better if this file was inside src/common/cuda instead of top level common

Reply:
Shouldn't ./operator/nn/cudnn/cudnn_deconvolution-inl.h be removed?  I don't believe it's referenced.